packages/mirinae/src/inputs/dropdown/select-dropdown/PSelectDropdown.vue

```diff
@@ -94,16 +91,23 @@ const state = reactive({
         }
         return res;
     }, {})),
+    innerText: computed<string>(() => {
+        // TODO: need to check at console for translation.
+        if (typeof (props.innerLabel) === 'string') {
+            return state.selectedItems?.length > 0
+                ? `${state.selectedItems[0].label || state.selectedItems[0].name || ''}`
+                : (props.innerLabel || 'Label');
+        }
+
+        return state.selectedItems.length > 0
+            ? (state.selectedItems[0].label || state.selectedItems[0].name || '')
+            : (props.placeholder || 'Select');
```

Comment:
Suggestion: It seems like 'Select' needs translation. The translations for the components are located under the mirinae package.

Reply:
Issue: It appears that there is a need for implementation regarding the case when innerLabel is of boolean type.